client/app/config/steps/password.coffee

```diff
@@ -7,16 +7,23 @@ module.exports = {
 
     # If OK, return null
     # if not return an Array of errors
-    validate: (data) ->
-        return null
+    # that will be triggered throw onboarding
+    # to dispatch error into app
+    validate: (data={}) ->
+        unless data.password?
+            return { type: 'password', text:'step empty fields'}
+        else
+            return null
+
 
     save: (data) ->
-        data = JSON.stringify data
         return new Promise((resolve, reject) ->
             jQuery.post({
```

Comment:
Suggestion: Would it be possible to use fetch (like here https://github.com/cozy/cozy-proxy/blob/development/client/app/config/steps/welcome.coffee#L6) instead of jQuery.post and maybe remove the jQuery dependance here? That's also good to keep coherence in our code.